### ISSUE TITLE
Add legal subpage or section

### ISSUE THREAD

**Issue by veroandreo** [OSGeo/grass-website#32]
Add a section or subpage maybe under *About* with the info currently in
- https://grass.osgeo.org/home/imprint
- https://grass.osgeo.org/home/copyright

**Comment by nbozon** [comment]
This is partly done: https://staging.grass.osgeo.org/about/credits/
but there is of course room for improvement here

**Comment by veroandreo** [comment]
Suggestion in the call today was to create a "Licence" page under [About](content/about) to add imprint and copyright info; and (to keep the number of items in the menu to a max of 5) move History to a sub-page or section in the Community page.

Another option I can think of now is to remove "News" from About and make it a separate menu (maybe the first one before Download) so we can add Licence and still keep History as a separate sub-menu in About. 

Opinions @lucadelu @neteler @nbozon ?

**Comment by lucadelu** [comment]
`News` are already in home page I think we could remove from `About`.
License page improvements in aa19522 and added to menu in 3f9937c

**Comment by hellik** [comment]
I think a licence page should be under _about_, IMHO it's important that a user should know about the licence how to use the software by 1 or maximal 2 clicks.

**Comment by hellik** [comment]
> 
> 
> This is partly done: https://staging.grass.osgeo.org/about/credits/
> but there is of course room for improvement here

in credits, the license is for the website, but not for the software.

**Comment by hellik** [comment]
>     * https://grass.osgeo.org/home/copyright

I think the basic information should be taken from:

https://grass.osgeo.org/home/copyright

I'll have a look how to improve and extend it for a starting point.

**Comment by hellik** [comment]
starting text may be:

```
GRASS GIS is distributed as Free Software under the terms of the Free Software Foundation’s [https://www.fsf.org/] GNU General Public License [https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html].

The General Public Licenses are designed to make sure that you have the freedom to distribute copies of free software, that you receive source code or can get it if you want it, that you can change the software or use pieces of it in new free programs.

This means that you have essential freedoms such as: Run the software for any purpose, find out how it works, make changes, redistribute copies (of modified versions).

GRASS GIS is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. 

The full licensing terms are available in the GPL.TXT [https://github.com/OSGeo/grass/blob/master/GPL.TXT] file.

If you are not sure about the differences between public domain software, Free Software and proprietary products, have a look at this graph describing 'Categories of Free and Non-Free Software' [https://www.gnu.org/philosophy/categories.en.html].

The software is Copyright by members of the GRASS Development Team. Parts of GRASS GIS are not copyright by the GRASS development team. The original authors hold the copyrights and you have to abide to their licensing terms where noted.
```

@veroandreo @neteler : where (in which subfolder of the website source) should I a do a PR for the text?

**Comment by veroandreo** [comment]
> `News` are already in home page I think we could remove from `About`.

Fully agreed! `News` should be a separate menu, it should not be inside `About`

**Comment by hellik** [comment]
> License page improvements added to menu in 3f9937c

@lucadelu 

looking at https://staging.grass.osgeo.org/ , the menue entry isn't there, see screenshot:

![license2](https://user-images.githubusercontent.com/12000843/79261502-3baff880-7e90-11ea-9bf8-cc21d8e6b889.PNG)

**Comment by veroandreo** [comment]
Closing this issue because it has been addressed in #122
